js/lib/ui.js

```diff
@@ -418,11 +418,23 @@ elgg.ui.initAccessInputs = function () {
 		}
 		var $select = $(this),
 			acl = $select.data('group-acl'),
-			$note = $('.elgg-input-access-membersonly', this.parentNode);
+			$note = $('.elgg-input-access-membersonly', this.parentNode),
+			$commentCount = $select.data('comment-count'),
```

Comment:
ok, I'm confused as to what you're suggesting with that.  Are you saying it should be:

    commentCount = $select.data('commentCount'),

If so it doesn't work, I just tried it.